### DISCUSSION TITLE
Add Reactions independantly?

### DISCUSSION THREAD

**Discussion by MrHinsh** [giscus/giscus#1554] [Q&A] [answered]
Hi @laymonage, is there a way to have the "reaction" added to a separate location?

I want to have the reaction UX added to the top of my post as well as the comments and reaction at the bottom!

**Comment by laymonage** [comment, marked as answer]
Not without embedding the widget twice. It's too much work to do efficiently as the widget has no control over the parent page. You're better off using GitHub's GraphQL API directly for such bespoke needs, at least for now.

**Reply by MrHinsh** [reply]
I see what you are doing... ill leave it then. I don't want to have to replicate all your logic to find the discussion.
